lib/src/widgets/utils/ds_card.widget.dart

```diff
@@ -78,6 +78,14 @@ class DSCard extends StatelessWidget {
       case DSMessageContentType.contact:
         return _buildContact();
 
+      case DSMessageContentType.reply:
+        return DSCard(
```

Comment:
You should pass all ```DSCard``` params, just to make sure that everything is going through the widgets.